README.md

```diff
@@ -168,7 +172,9 @@ Or, if you use Maven:
 
 ## Relevant Links
 * [skUnity forums](https://forums.skunity.com)
-* [Add-on releases at skUnity](https://forums.skunity.com/forums/addon-releases)
+* [skUnity Documentation](https://docs.skunity.com)
```

Comment:
You should link to the official Skript docs instead.

Reply:
Skript official docs already have its own section above, so the skUnity docs should be just removed

Reply:
While I it's still link it might be good to add it there as well or see if there's anymore information that could be added to the section, It's basically a one liner and just like help us test isn't noticeable